# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/moz.build
# Commit: 7c4b003d1e48
# Full Hash: 7c4b003d1e48ac2e847191fe36ce4c39b9280d54
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-18 04:27:40
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 4: Implement the third-party cookie blocking excpetion in CookieService. r=cookie-reviewers,baku,valentin
#   
#   The patch integrates the third-party cookie blocking excpetion in
#   CookieService.
#   
# ==============================================================================

diff -r 36163490219b -r 7c4b003d1e48 netwerk/cookie/moz.build
--- a/netwerk/cookie/moz.build	Tue Dec 17 21:44:37 2024 +0000
+++ b/netwerk/cookie/moz.build	Tue Dec 17 21:44:37 2024 +0000
@@ -34,6 +34,7 @@
     "CookieServiceChild.h",
     "CookieServiceParent.h",
     "CookieStorage.h",
+    "ThirdPartyCookieBlockingExceptions.h",
 ]
 UNIFIED_SOURCES += [
     "Cookie.cpp",
@@ -48,6 +49,7 @@
     "CookieServiceChild.cpp",
     "CookieServiceParent.cpp",
     "CookieStorage.cpp",
+    "ThirdPartyCookieBlockingExceptions.cpp",
 ]
 XPCSHELL_TESTS_MANIFESTS += [
     "test/unit/xpcshell.toml",